nan_typedarray_contents.h

```diff
@@ -31,9 +31,14 @@ class TypedArrayContents {
       v8::Local<v8::ArrayBuffer> buffer = array->Buffer();
 
       length = byte_length / sizeof(T);
+#if (V8_MAJOR_VERSION >= 10 ||                                                 \
+     (V8_MAJOR_VERSION == 10 &&                                                \
+      (defined(V8_MINOR_VERSION) && V8_MINOR_VERSION >= 5)) ||                 \
+       (NODE_MAJOR_VERSION == 18 && NODE_MINOR_VERSION >= 8))
```

Comment:
I think this means a native addon built with 18.8.0 or newer wont work with older 18 node.js versions.
Does this fit the NAN compatibilty schema?
Maybe go for >= 20?